nestkernel/connection.h

```diff
@@ -305,39 +374,45 @@ class Connection
    * \param the last spike produced by the presynaptic neuron (for STDP and
    * maturing connections)
    */
-  void check_connection_( Node& dummy_target, Node& source, Node& target, const size_t receptor_type );
+  void check_connection_( Node& dummy_target,
+    Node& source,
+    Node& target,
+    const synindex syn_id,
+    const size_t receptor_type );
 
-  // The order of the members below is critical as it influcences the size of the object.
-  // Please leave unchanged!
   targetidentifierT target_;
-  // syn_id (9 bit), delay (21 bit) in timesteps of this connection and more_targets and disabled flags (each 1 bit)
-  SynIdDelay syn_id_delay_;
+  bool more_targets_ : 1;
+  bool disabled_ : 1;
+  DelayTypeT delay_;
+  // There are still 14 bits to spare here. If more bits are required, the sizes of the delays in the delay struct could
+  // be reduced even more as well.
```

Comment:
I don't understand this comment. The full assumed size of the `Connection` is not mentioned anywhere, nor is the size of `targetidentifierT`. Details on the construction here could go into the doxygen comment for the class.